MH4U Dex/QuestDetailsTableViewController.m

```diff
@@ -140,10 +171,21 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
 {
     UIStoryboard *storyBoard = [UIStoryboard storyboardWithName:MHDStoryBoardIdentifier.main bundle:nil];
     MonsterContainerViewController *monsterVC = [storyBoard instantiateViewControllerWithIdentifier:NSStringFromClass([MonsterContainerViewController class])];
+    RegionContainerViewController *regionVC = [storyBoard instantiateViewControllerWithIdentifier:NSStringFromClass([RegionContainerViewController class])];
     switch (indexPath.section) {
         case DetailsSection:
-            //Do nothing.
-            return;
+            if (indexPath.row != 11) {
```

Comment:
I think here if you have the `if` case handle `(indexPath.row == 11)` the `else` case can be removed entirely and you can just `return` at the end. 

A strategy to keep in mind is that when there's an `if/else` where there's one unusual case and a bunch of standard cases, handle the more unusual case in the `if` and then move on to the default handling in the `else`.